language/return_spec.rb

```diff
@@ -252,6 +252,15 @@ def f
 
   ruby_version_is '2.4' do
     describe "at top level" do
+      before :each do
+        @filename = tmp("top_return.rb")
+        ScratchPad.record []
+      end
+
+      after do
+        rm_r @filename
+      end
+
       it "stops file execution" do
         ruby_exe(<<-END_OF_CODE).should == "before return\n"
```

Comment:
@eregon
I left `ruby_exe` here intentionally to test terminating the ruby process

Reply:
Good idea, it's worth testing this also works for the main script.